README.md

```diff
@@ -1,2 +1,23 @@
 # snek
-If you hit yourself or the wall, you'll die. If you eat the fruit (F) your score increases and tail gets longer.
+snek - A simple snake in the console
```

Comment:
Not anyone knows the game "Snake", so it would be nice to explain it.

Reply:
yes, i add that to the TODO, the README has to be reworked anyway with proper explanation, of the compiling with dependencies etc and probably later some extra things